guidelines/frontend-coding-standard/double-not-operator.md

```diff
@@ -0,0 +1,96 @@
+# Use `!!` to cast anything into boolean
+
+It is good to use double NOT `!!` when you want to ensure that a return value is always boolean type in methods that returns a boolean value based on some condition.
```

Comment:
Nice, but let's agree to use stronger wording here.
instead of `It's good to use`, try:

* Suggest using double not `!!` when we are expecting a boolean value in a expression`

Plus provide some example, sth similar to this:

```js
const { data } = useEntity(id);
// 🤔 avoid this
if (data) {
  // do something
}
// 👍🏻 Suggest this
if (!!data) {
  // do something
}
```